pandunia-lekse.csv

```diff
@@ -1351,6 +1376,7 @@ kankexe||||eyelash||pestaña||||||||||||||silmäripsi|rzęsa
 kanowe|||eng:canoe, fra:canoë, tur:kano, spa:por:canoa, rus:каноэ (kanoe), jpn:カヌー (kanū), kor:카누 (kanu)|canoe|canoë|canoa|canoa|каноэ|||カヌー|카누||||||kano|kanuo (kanoto)|kanootti|kanoe
 kansuye|||tam:கண்ணீர் (kaṇṇīr), mal:കണ്ണുനീർ (kaṇṇunīr), tel:(kannīru), kor:눈물 (nunmul), vie:nước mắt, may:air mata, tha:น้ำตา|tear drop||lágrima||||||||||||||kyynel|łza
 kansuygase||||tear gas|gaz lacrymogène|gas lacrimógeno|gás lacrimogéneo|слезоточи́вый газ||催泪瓦斯|催涙ガス||hơi cay||||||||gaz łzawiący
+kantalupi melone|||eng:cantaloupe, spa:por:cantalupo, fra:cantaloup, deu:Cantaloup, rus:канталупа (kantalupa), jpn:カンタロープ (kantarōpu)|cantaloupe||melón cantalupo|meloa (cantalupo)|канталупа||鈔皮瓜|カンタロープ||||||||||
```

Comment:
It bothers me that the word "kantalup-" doesn't exist on its own.

Reply:
That's fair. I was reluctant to make it its own noun since it's just a variety of a subspecies of muskmelons, and many languages don't make any distinction between them and other muskmelons as far as I can tell. It makes sense that if it has its own root it might as well have its own noun, though.

Reply:
Alternatively, we could make "kantalupe" a city name (there are several Italian cities of that name) and thus preserve the etymology of the fruit.

Reply:
There are so many varieties of fruit. I'm not really able to judge all names but I can only express my wish that we would use as much of normal Pandunia as possible.

Reply:
I think it makes sense as long as "kantalupe" is in the dictionary as "Cantalupo" to call it "kantalupomelone". the only translations I can find for "cantaloupe" that aren't just the word for muskmelon are variations on that city name.